src/components/NewCommentForm.tsx

```diff
@@ -75,25 +176,44 @@ export const NewCommentForm: React.FC = () => {
             id="comment-body"
             name="body"
             placeholder="Type comment here"
-            className="textarea is-danger"
+            className={cn('input', {
+              'is-danger': isBodyError,
+            })}
+            value={body}
+            onChange={event => {
+              setBody(event.target.value);
+              setIsBodyError(false);
```

Comment:
it is bad practise to make anonymous arrow functions in jsx.
make handler in component and pass reference here